components/dashboard/src/workspaces/Workspaces.tsx

```diff
@@ -162,8 +162,8 @@ export default function () {
                                                 </span>
                                             </div>
                                             <div className="text-sm flex-auto">
-                                                Unpinned workspaces that have been inactive for more than 14 days will
-                                                be automatically deleted.{" "}
+                                                Workspaces that were stopped more than 24 hours ago. Note that these
+                                                will be automatically deleted after 14 days.{" "}
```

Comment:
**suggestion**: In the spirit of using plain language with simpler copy and messaging that also favor brevity, we could rephrase this like the following, unless there are any strong feelings about keeping it as is. [WDYT](https://en.wiktionary.org/wiki/WDYT)?

> Workspaces that have been stopped for more than 24 hours. Inactive workspaces are automatically deleted after 14 days. 

**thought**: I wouldn't mind keeping the _Unpinned_ prefix to be more explicit, but we can remove it out for now and improve later in https://github.com/gitpod-io/gitpod/issues/10921, as changes here are already a good MVC improvement. Cc @jldec 

> Unpinned workspaces that have been stopped ... 

To clarify more and in case it helps, here's how I read this copy.

<img width="973" alt="copy" src="https://user-images.githubusercontent.com/120486/176013378-31d4bcf6-5016-47c9-a173-8cdd56eb9a3d.png">

Reply:
Alternative suggestion to avoid line break on expansion, etc.

> Unpinned workspaces that are stopped for more than 24 hours, automatically deleted after 14 days.